worlds/empty.world

```diff
@@ -1,7 +1,11 @@
 <?xml version="1.0" ?>
 <sdf version="1.5">
   <world name="default">
-    <!-- A global light source -->
+    <scene>
+      <ambient>0.95 0.95 0.95 1</ambient>
+      <background>0.3 0.3 0.3 1</background>
+      <shadows>true</shadows>
+    </scene>
```

Comment:
I do appreciate you fixing the lighting in gazebo, but I think this would have been better if it belonged to a separate PR